### DISCUSSION TITLE
Is YARN a pre-req for Airflow install ?

### DISCUSSION THREAD

**Discussion by krishraj360** [apache/airflow#26983] [Q&A]
Hi , 
I am new to Airflow and trying to install and test it. So, please excuse me if my understanding is wrong. My usecase is to see if it can completely replace Oozie in my hadoop cluster. Upon further reading, I understand that Airflow DAGs consists of tasks which run on different type of executors ( either local or distributed such as celery, kubernetes ). 

1. If tasks run on these executors, what task would run in YARN containers if I trigger a Hive batch job ?
2. Is YARN not necessary for Airflow to work ? 
3. Is YARN needed only if I run Hadoop specific jobs ?

**Comment by potiuk** [comment]
Yarn is only need for Webserver asset compilation. And if you are using airlfow released packages (which you should) yarn is not needed. So I am not sure your question has any sense at all.
